rotkehlchen/chain/ethereum/airdrops.py

```diff
@@ -382,6 +409,8 @@ def check_airdrops(
     data_dir = database.user_data_dir.parent
     airdrop_tuples = []
     for protocol_name, airdrop_data in AIRDROPS.items():
+        if airdrop_data.cutoff_time is not None and time() > airdrop_data.cutoff_time:
+            continue
```

Comment:
don't execute the function on every loop iteration. Python has to move the context and is expensive. Create a variable outside and assign it to time(). Also shouldn't you use `ts_now()`

Reply:
Can you also add a log debug here saying that you skip the airdrop?